src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// Write code here
+import React from 'react';
+
+export const Person = ({ personObj }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${personObj.name}`}
+    </h2>
+
+    {personObj.age && (
+      <>
+        <p className="Person__age">
+          {`I am ${personObj.age}`}
+        </p>
+      </>
+    )}
+
+    <p className="Person__partner">
+      { personObj.isMarried
+        ? `My ${personObj.sex === 'm' ? 'wife\'s' : 'husband\'s'} name is ${personObj.partnerName}`
+        : 'I am not married' }
```

Comment:
nested ternary operators is not good practice. Move to function or anything else solution